src/anonymous-purchase-consequences-screen/AnonymousPurchaseConsequencesScreenComponent.tsx

```diff
@@ -29,9 +26,12 @@ export const AnonymousPurchaseConsequencesScreenComponent = ({
   leftButton,
 }: Props) => {
   const styles = useStyle();
+  const { theme } = useTheme();
   const {t} = useTranslation();
   const focusRef = useFocusOnLoad();
 
+  const themeColor = theme.Background.Accent[0];
```

Comment:
In cases like this, `themeColor` should be passed in as an argument to `useStyle`. See e.g. `useStyles` in `TravelTokenBox.ts` for reference.